test/dlang/d2-only/Makefile

```diff
@@ -15,7 +15,7 @@ endif
 all:
 	echo 'We just made "all"' > make-all.stamp
 
-unittest integrationtest:
+test:
```

Comment:
This looked unrelated but I get no that this is because coverage gets enabled for all targets

Reply:
Yes, before the `dlang make` command ran `unittest` and `integrationtest` targets separately so reports could potentially be sent separately to codecov (this never actually worked because `integrationtest` reports of different tests will overwrite themselves, and in a racy way, so the last executed report was sent and maybe it changed from build to build, changing coverage results).

Now we are back to just calling the `test` target only for tests and letting `dmd` do its magic at merging reports.